### PR TITLE
OUR-318: Fix text when only on vote for a package

### PR DESCRIPTION
**Pull request by Aaen**
When a package only has 1 vote the text should be "1 vote" instead of "1 votes".

Some examples:
https://our.umbraco.org/projects/collaboration/mdedit/
https://our.umbraco.org/projects/backoffice-extensions/nupickers-vimeo-picker/
https://our.umbraco.org/projects/backoffice-extensions/attackmonkey-grid-locker/

### CODE REVIEW

File: OurUmbraco.Site/Views/MacroPartials/Projects/ProjectDetails.cshtml [hunk 1]
```diff
@@ -51,15 +51,48 @@
...
+               }
+               else
+               {
+                <span>@totalVotes vote</span>
+               }
+           }
+
+            @if (Members.IsLoggedIn() && !hasMemberVoted)
+            {
+                if (totalVotes == 0 || totalVotes > 1)
```
Comment (nul800sebastiaan) [inline OurUmbraco.Site/Views/MacroPartials/Projects/ProjectDetails.cshtml]: You're repeating the exact same code 3 times here. Better to just make one variable like:
```
var voteText = "votes";
if(totalVotes == 1) { voteText = "vote"; } 
```
And then use `@voteText` everywhere.

**Comment by Aaen** [comment]
I added the variable as you suggested so we got a much nicer solution

**Comment by bjarnef** [comment]
L46-47 can also just be written in a single line:

vote / votes
```
var voteText = totalVotes == 1 ? "vote" : "votes";
```

or combine it with the number of votes:

1 vote / 2 votes
```
var voteText = totalVotes + " " + ((totalVotes == 1) ? "vote" : "votes");
```

**Comment by nul800sebastiaan** [comment]
I just merged @greystate 's in, whoops didn't see this was a duplicate! 
https://github.com/umbraco/OurUmbraco/pull/71

**Pull request closed without merging**